codespell_lib/data/dictionary.txt

```diff
@@ -17698,6 +17702,8 @@ seleted->selected
 self-contianed->self-contained
 self-referencial->self-referential
 self-refering->self-referring
+selt->set
```

Comment:
This could also be self and sett as in a badgers house.

In fact where did these both come from, were they real-world? I can't quite work out on a Qwerty keyboard how the l gets involved given where the other keys are.